src/App.js

```diff
@@ -1,11 +1,32 @@
-import React from 'react';
-
-function App() {
-  return (
-    <h1>
-      Hello Wors
-    </h1>
-  );
+import React, { Component } from 'react';
+
+import Header from './Components/Header/Header';
+import Form from './Components/Form/Form';
```

Comment:
components folder should start with lower case,
in React, with a great letter, you name only component files